lte/gateway/c/core/oai/tasks/mme_app/mme_config.c

```diff
@@ -348,9 +361,9 @@ void create_partial_lists(mme_config_t* config_pP) {
   /* Copy TAIs from served_tai to partial lists. If there are more that 16 TAIs,
    * add the TAIs to a new partial list
    */
-  uint8_t size = config_pP->served_tai.nb_tai > MAX_TAI_SUPPORTED ?
-                     MAX_TAI_SUPPORTED :
-                     config_pP->served_tai.nb_tai;
+  uint8_t size            = config_pP->served_tai.nb_tai > MAX_TAI_SUPPORTED ?
```

Comment:
not related to your changes, can we update this to this...?
```suggestion
  uint8_t served_tai_size            = config_pP->served_tai.nb_tai > MAX_TAI_SUPPORTED ?
```